$NetBSD: patch-aa,v 1.17 2008/02/20 10:00:24 markd Exp $

utmpx patch gets ktalkd to build on Solaris 10 but wont actually
read the file properly - needs to be fixed.

--- configure.orig	2006-10-03 07:24:51.000000000 +1300
+++ configure
@@ -44334,6 +44340,7 @@ else
 	/var/run/utmp \
 	/var/adm/utmp \
 	/etc/utmp     \
+	/var/adm/utmpx \
      ; \
     do
      if test -r "$ac_file"; then
